### PR TITLE
Add TradingView.app version 1.0.0-beta.1

### PR DESCRIPTION
**Pull request by xel-arjona**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/tradingview-beta.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "tradingview-beta" do
+  version "1.0.0-beta.1"
+  sha256 :no_check
+
+  url "https://tvd-packages.tradingview.com/beta/#{version}/win32/x64/TradingView.dmg"
+  name "tradingview-beta"
+  desc "TradingView Desktop Beta"
```
Comment (reitermarkus) [inline Casks/tradingview-beta.rb:7]: This is not a description, this should be the name.

Comment (xel-arjona) [inline Casks/tradingview-beta.rb:7]:
```suggestion
  desc "TradingView Desktop (Beta) application based on Electron."
```

File: Casks/tradingview-beta.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cask "tradingview-beta" do
+  version "1.0.0-beta.1"
+  sha256 :no_check
+
+  url "https://tvd-packages.tradingview.com/beta/#{version}/win32/x64/TradingView.dmg"
+  name "tradingview-beta"
```
Comment (xel-arjona) [inline Casks/tradingview-beta.rb:6]:
```suggestion
  name "TradingView Desktop Beta"
```

**Comment by miccal** [comment]
Since this is only available (as this stage) as a beta, this should be submitted to the main `homebrew-cask` repository.

Please re-submit this to the main repository, thanks.

**Pull request closed without merging**